krux/parser.py

```diff
@@ -205,14 +205,14 @@ def get_group(parser, group_name, env_var_prefix=None):
 
 
 class KruxParser(ArgumentParser):
-    def add_argument_group(self, env_var_prefix, *args, **kwargs):
+    def add_argument_group(self, *args, **kwargs):
```

Comment:
In this method, we never called `super()`. Sifting through `argparse` code, that would be a call to `argparse._ActionsContainer.add_argument_group(self, *args, **kwargs)`. If that’s intentional, it’s worth mentioning.